### ISSUE TITLE
1.0.0 release is almost a year old

### ISSUE THREAD

**Issue by clausherther** [Montreal-Analytics/dbt-gloss#62]
Seems like the latest release over 11 months old and doesn't include many of the newer tests, such as `check-model-name-contract`, resulting in 
```
[ERROR] `check-model-name-contract` is not present in repository https://github.com/offbi/pre-commit-dbt.  Typo? Perhaps it is introduced in a newer version?
```

The current fix is to pin to `main`, which pre-commit only sort of supports: 
```
The 'rev' field of repo 'https://github.com/offbi/pre-commit-dbt' appears to be a mutable reference (moving tag / branch).  Mutable references are never updated after first install and are not supported.
```

Any plans for a future release?

**Comment by Aylr** [comment]
@clausherther I'm using a workaround by pinning to a specific commit. It isn't as pretty as a release number but gets the job done.

```
uses: offbi/pre-commit-dbt@cc144b9a1a24112428d9a52a66905815990e4554
```

@tomsej are you open to support maintaining this excellent package?

**Comment by katieclaiborne** [comment]
Do these fixes apply to the `check-model-parents-database` hook, as well?

They work for my teammate, who is testing `check-model-name-contract`, but when we add `check-model-parents-database`, the error from Claus's description above reappears.
```
Error:  `check-model-parents-database` is not present in repository https://github.com/offbi/pre-commit-dbt.  Typo? Perhaps it is introduced in a newer version?  Often `pre-commit autoupdate` fixes this.

**Comment by michael-cummings-cko** [comment]
@katieclaiborne I am seeing the same error when using the version of pre-commit-dbt (cc144b9a1a24112428d9a52a66905815990e4554).

Is this hook working as expected for other?

**Comment by katieclaiborne** [comment]
We're working around the issue by pointing our `.pre-commit-config.yaml` file to Kyle's branch in #66.

**Comment by JFrackson** [comment]
v1.1.0 has been released recently and v1.1.1 with some small patches should be coming soon. Sorry for the delay!
